iothub/device/src/Transport/AmqpClientConnectionPool.cs

```diff
@@ -5,181 +5,117 @@ namespace Microsoft.Azure.Devices.Client.Transport
 {
     using System;
     using System.Collections.Generic;
-    using System.Text;
-    using System.IO;
-    using System.Collections.Concurrent;
     using Microsoft.Azure.Devices.Shared;
+    using System.Threading;
+    using System.Threading.Tasks;
 
-    delegate void RemoveClientConnectionFromPool(DeviceClientEndpointIdentity deviceClientEndpointIdentity);
+    delegate void OnClientConnectionIdle(AmqpClientConnection amqpClientConnection);
 
     internal class AmqpClientConnectionPool
     {
-        ConcurrentDictionary<DeviceClientEndpointIdentity ,AmqpClientConnection> deviceConnectionDictionary;
-        ConcurrentBag<AmqpClientConnection> connectionPool;
-
-        ConcurrentDictionary<DeviceClientEndpointIdentity, AmqpClientConnection> deviceConnectionDictionaryIoTHubSas;
-        ConcurrentBag<AmqpClientConnection> connectionPoolIotHubSas;
-
-        AmqpClientConnectionFactory amqpClientConnectionFactory;
+        private static readonly TimeSpan TimeWait = TimeSpan.FromSeconds(10);
+        private ISet<AmqpClientConnectionMux> AmqpClientSasConnections;
```

Comment:
Can you define magic number somewhere.

Reply:
`TimeWait` is already `static` and it's specialized for `AmpqClientConnectionPool`. I believe this is the best place to put this constant.